packages/atlas/src/components/_ypp/YppAuthorizationStepModal/YppAuthorizationStepModal.tsx

```diff
@@ -0,0 +1,140 @@
+          title: 'Select channel',
+          description: 'Select the Atlas channel you want your YouTube channel to be connected with.',
+          primaryButton: { text: 'Select channel' },
+          component: <SelectChannelDialog channels={channels} />,
+        }
+      case 'requirements':
+        return {
+          title: 'Requirements',
+          description:
+            'Before you can apply to the program, make sure both your Atlas and YouTube channels meet the below conditions.',
+          primaryButton: {
+            text: 'Authorize with YouTube',
+          },
+          component: <RequirementsDialog />,
+        }
+      case 'details':
+        return {
+          title: 'Details',
+          description: 'We need your email address to send you payment information. No spam or marketing materials.',
+          primaryButton: {
+            text: 'Continue',
+          },
+          component: <DetailsFormDialog />,
+        }
+      case 'terms-and-conditions':
+        return {
+          title: 'Terms & conditions',
+          description:
+            'Once automatic YouTube videos sync is available, in order for it to work, your Atlas channel [NEEDS TO DO WHAT?]. This is purely a technical measure and does not affect ownership and rights to the content uploaded to you Atlas channel.',
+          primaryButton: { text: 'Accept terms & sign' },
+          additionalSubtitleNode: (
+            <AdditionalSubtitle>
+              <Text variant="h400" as="span">
+                Automatic YouTube sync
+              </Text>{' '}
+              <Text variant="t100" as="span" color="colorText">
+                Coming later this year
+              </Text>
+            </AdditionalSubtitle>
+          ),
+          component: <TermsAndConditionsDialog />,
+        }
+      case 'summary':
+        return {
+          title: 'Congratulations!',
+          description: (
+            <>
+              Congratulations! You just received 200 JOY as your Atlas channel is now officially enrolled in the YouTube
+              Partner Program and tied with a YouTube channel.{' '}
+              <Text variant="t200" as="p" margin={{ top: 2 }} color="colorText">
+                All information around your activity in the program can be found in the{' '}
+                <Button variant="primary" _textOnly>
+                  YPP page in Studio
+                </Button>
+                .
+              </Text>
+            </>
+          ),
+          primaryButton: { text: 'Go to dashboard' },
+          component: <Img src={appScreenshot} />,
+        }
+      case 'connect-with-yt':
+        return {
+          title: 'Connect with YouTube?',
+          description:
+            'Reupload and backup your YouTube videos to receive to receive a guaranteed payout in the YouTube Partner Program.',
+          primaryButton: {
+            text: 'Authorize with YouTube',
+          },
+        }
+    }
+  }, [channels, step])
```

Comment:
Text in the secondary button should be `Close` not `Back`. The icons in the dialog are different as well: 
![image](https://user-images.githubusercontent.com/51168865/190172575-a7754f10-fe87-40ce-8091-814b24755097.png)